plugins/candela/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "candela-plugin",
     "version": "0.1.0",
     "dependencies": {
-        "candela": "^0.12.1",
+        "candela": "^0.16.0",
```

Comment:
We do this even pre-1.0 packages? Caret treats 0.x differently. From the description of caret,

> Many authors treat a 0.x version as if the x were the major "breaking-change" indicator.

This has been true with Candela. If we changed Candela to tilde, it would break if we make more changes before 1.0.

Alternately we could pin Candela to a specific version, then use tilde when it reaches 1.0.